OptimoveSDK/Sources/Classes/Optimove.swift

```diff
@@ -22,6 +22,7 @@ typealias Logger = OptimoveCore.Logger
     }()
 
     private let container: Container
+    private var config: OptimoveConfig!
```

Comment:
This maybe isn't the neatest way to do this, but I couldn't see how to easily provide any arguments to the shared initialiser block.

As usage is very constrained, perhaps it's acceptable.

Reply:
I can live with this, its easier than making nullable and checking every time we try to use it.